build.gradle

```diff
@@ -113,9 +113,11 @@ dependencies {
     // Include all jar-files in the 'lib' folder as dependencies
     implementation fileTree(dir: 'lib', includes: ['*.jar'])
 
-    implementation 'org.apache.pdfbox:pdfbox:3.0.1'
-    implementation 'org.apache.pdfbox:fontbox:3.0.2'
-    implementation ('org.apache.pdfbox:xmpbox:3.0.2') {
+    // 3.0.2 is buggy!
+    def pdfbox = "3.0.1"
```

Comment:
@Siedlerchr Do we really use variables here? This is the first time, I saw it. Does this work with dependabot? Shouldn't we add the exception to https://github.com/JabRef/jabref/blob/8ff0ae6927ebc53081960c1f78a7470013aed4b4/.github/dependabot.yml#L9 - to enable update to 3.0.3 or later versions?

Reply:
https://github.com/dependabot/dependabot-core/issues/1618#issuecomment-665863295

Did this for Lucene as well.  Should work. 
Good idea to add an exception to the yaml file

Reply:
I remember maven files where in another file the versions were set. Or even in a dependency 🤣🤣

Good that we keep it close to the use 😅